.changeset/rich-moles-compare.md

```diff
@@ -0,0 +1,6 @@
+---
+"@nextui-org/use-clipboard": patch
+"@nextui-org/snippet": patch
```

Comment:
not required (?)

Reply:
if the use-clipboard upgrade to 2.1.2 then the latest @nextui-org/snippet is using 2.1.1 will it upgrades too? or we can delete it

Reply:
`@nextui-org/snippet` has a dependency on `use-clipboard`. When `use-clipboard` got bumped, `@nextui-org/snippet` will be also bumped automatically by changeset.